meson.build

```diff
@@ -42,11 +42,13 @@ deps = [dependency('jsoncpp'),
 oomd_lib = library('oomd',
                    srcs,
                    include_directories : inc,
+                   install: true,
                    dependencies : deps)
 executable('oomd_bin',
            files('Main.cpp'),
            include_directories : inc,
            dependencies : deps,
+           install: true
            link_with : oomd_lib)
```

Comment:
It looks like meson installs `liboomd.so` into `/usr/local/lib64/liboomd.so`. This path isn't in my LD_LIBRARY_PATH. Hard coding in an rpath should be a tenable solution.

Something like this should work:
```
+           install: true,
+           link_with : oomd_lib,
+           install_rpath: join_paths(get_option('prefix'), get_option('libdir')))
```